app/lib/frontend/templates/views/pkg/admin_page.mustache

```diff
@@ -48,32 +48,52 @@
   {{/user_has_publisher}}
 </div>
 
-<h2>Discontinued</h2>
-{{^is_discontinued}}
+<h2>Package Options</h2>
+
+<h3>Discontinued</h3>
 <div>
-  <p>
-    If no one is maintaining this package, you can mark it as discontinued.
-    Discontinued packages remain available, but pub.dev doesn’t display them in search results.
-  </p>
-  <p>
-    <button
-      id="-admin-is-discontinued-toggle"
-      class="mdc-button mdc-button--raised"
-      data-mdc-auto-init="MDCRipple">Mark as "discontinued"</button>
-  </p>
+  If no one is maintaining this package, you can mark it as discontinued.
+  Discontinued packages remain available, but pub.dev doesn’t display them
+  in search results and won't analyze it for scoring.
+</div>
+<div class="mdc-form-field">
+  <div class="mdc-checkbox">
+    <input type="checkbox"
+           class="mdc-checkbox__native-control"
+           id="-admin-is-discontinued-checkbox"
+           {{#is_discontinued}}checked="checked"{{/is_discontinued}}
+    />
+    <div class="mdc-checkbox__background">
+      <svg class="mdc-checkbox__checkmark" viewBox="0 0 24 24">
+        <path class="mdc-checkbox__checkmark-path" fill="none" d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
+      </svg>
+      <div class="mdc-checkbox__mixedmark"></div>
+    </div>
+    <div class="mdc-checkbox__ripple"></div>
+  </div>
+  <label for="-admin-is-discontinued-checkbox">Mark "discontinued"</label>
 </div>
-{{/is_discontinued}}
-{{#is_discontinued}}
+
+<h3>Unlisted</h3>
 <div>
-  <p>
-    If you want to continue developing this package, you can remove the discontinued status.
-    The package will be analyzed and included in search results.
-  </p>
-  <p>
-    <button
-      id="-admin-is-discontinued-toggle"
-      class="mdc-button mdc-button--raised"
-      data-mdc-auto-init="MDCRipple">Remove "discontinued"</button>
-  </p>
+  If you don't want this package to be discovered, you can mark it unlisted.
+  Unlisted packages remain available, they are getting analyzed for scoring,
+  but pub.dev doesn’t display them in search results.
```

Comment:
Maybe something like:
```html
A package can be marked as <i>unlisted</i> to avoid crowding search results.
Unlisted packages remain publicly available, but they will not show up in search results on pub.dev
with default search filters (<i>unlisted packages</i> can still be found using advanced search options).
```

IMO, the use-case is: to avoid crowding search results. But we could give examples of when this might be useful. Typically, it's when publishing a package that only used as transitive dependency, for example a package that contains libraries used internally by other packages, but that you don't think people should be using directly.

Perhaps it's better to document this and give examples on a page like: `https://dart.dev/tools/pub/publishing#unlisted`
Filed https://github.com/dart-lang/site-www/issues/2647 to track this.

@kwalrath if you have any quick suggestions for explanatory text feel free to leave us some wisdom :D

This is the "bla bla bla" section from the mockup in go/is:unlisted-on-pub